ui/src/components/WMWorkerPoolEditor/index.jsx

```diff
@@ -377,13 +415,26 @@ export default class WMWorkerPoolEditor extends Component {
                 requiresAuth
                 tooltipOpen
                 icon={<DeleteIcon />}
-                onClick={this.handleOnClick}
+                onClick={onDialogActionOpen}
                 tooltipTitle="Delete"
                 className={classes.deleteIcon}
                 disabled={actionLoading}
               />
             </SpeedDial>
           )}
+          {dialogOpen && (
+            <DialogAction
+              open={dialogOpen}
+              onSubmit={this.handleOnClick}
+              onComplete={onDialogActionComplete}
+              onClose={onDialogActionClose}
+              onError={onDialogActionError}
+              error={dialogError}
+              title="Delete Worker Pool?"
+              body={<Typography>This will delete the Worker Pool.</Typography>}
```

Comment:
It would probably be a good idea to mention the name of the worker pool we are trying to delete.